src/picongpu/include/particles/bremsstrahlung/SynchrotronFunctions.hpp

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright 2015 Heiko Burau
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "types.h"
+#include "cuSTL/container/HostBuffer.hpp"
+#include "cuSTL/container/HostBuffer.hpp"
+#include "cuSTL/cursor/Cursor.hpp"
+#include "cuSTL/cursor/navigator/PlusNavigator.hpp"
+#include "cuSTL/cursor/tools/LinearInterp1D.hpp"
+#include "cuSTL/cursor/BufferCursor.hpp"
+#include "simulation_defines.hpp"
+#include <boost/shared_ptr.hpp>
+#include <boost/array.hpp>
+#include <boost/numeric/odeint.hpp>
```

Comment:
please answer my question in https://github.com/ComputationalRadiationPhysics/picongpu/pull/1299#discussion_r47224204